tests/src/Unit/AuthenticatedClientTest.php

```diff
@@ -210,6 +211,32 @@ public function testGetConfig()
         $this->assertEquals('the-value', $authenticatedClient->getConfig('the-option'));
     }
 
+    /**
+     * Test setting a duration for renewed tokens.
+     *
+     * @covers ::setTokenDuration()
+     * @covers ::mergeAuth()
```

Comment:
I don't see how this is covering ::mergeAuth

Reply:
As mergeAuth is private, it's covered through the public method call on `request`. If you generate code coverage for this test you should see mergeAuth() being called.

Reply:
:+1: